File: aws/bedrock/node/ask-anthropic.js
import { converse } from "./bedrock-common.js";

const modelId =
  process.env.ANTHROPIC_BEDROCK_MODEL_ID || "us.anthropic.claude-sonnet-4-20250514-v1:0";

console.log(`Model: ${modelId}\n`);
console.log(await converse(modelId));

File: aws/bedrock/python/ask_anthropic.py
import os

from bedrock_common import converse


MODEL_ID = os.getenv(
    "ANTHROPIC_BEDROCK_MODEL_ID",
    "us.anthropic.claude-sonnet-4-20250514-v1:0",
)


def main() -> None:
    print(f"Model: {MODEL_ID}\n")
    print(converse(MODEL_ID))


if __name__ == "__main__":
    main()
